reference/OpenAPI.json

```diff
@@ -14197,86 +14268,21 @@
           }
         ]
       },
-      "19_res_validations": {
+      "ValidationsValidateResponseBody": {
```

Comment:
Jakoś dziwnie mi diffy pokazuje. Ale generalnie jak znajdziesz sobie `ValidationsValidateRequestBody`. To wydaje mi się, że brakuje tam jeszcze jednego property - `tracking_id`